### PR TITLE
Create encyclopdia entry on What is CSV

### PR DESCRIPTION
**Pull request by JeremyBarbosa**
### Description

Create an ecyclopedia entry on "What is CSV".

**NOTE: This was marked as important for SEO so I am unsure what additional work or review may be necessary**

### Type of Change

- [x] Adding a new entry

### Checklist

- [x] My entry follows the Codecademy Docs style guide
- [x] I have performed a self-review of my own writing and code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have checked my entry and corrected any misspellings
- [x] All writings are my own

### CODE REVIEW

File: content/general/concepts/what-is-csv/what-is-csv.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+Animal,Average Length (Feet),Average Weight (Tons)
+African Bush Elephant,19.7,5.4
+Indian Rhinoceros,13.8,2.1
+Hippopotamus,16.5,2.0
+Giraffe,16.9,1.1
+Gaur,12.5,1.0
+```
+
+| Animal | Average Length (Feet) | Average Weight (Tons) |
+|--|--|--|
```
Comment (sonnynomnom) [inline content/general/concepts/what-is-csv/what-is-csv.md:31]: Let's add spaces:

`| --- | --- | --- |`

File: content/general/concepts/what-is-csv/what-is-csv.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+Tags:
+  - "Files"
+Catalog Content:
+  - "https://www.codecademy.com/learn/paths/code-foundations"
+  - "https://www.codecademy.com/learn/paths/computer-science"
+---
+
+Comma-Seperated Values (CSV) is a text file format where individual values are seperated by commas. It is normally used to store tabular data and is favored for its simple structure allowing for straightforward parsing and data exchange.
+
+To store tabular data, CSV files usually represent each line as a new row within the table and the values delimited by commas as the cells within each row. However, this practice is not formalized so the interpretation of the cell's contents is normally application-specific. For example, some applications use quotation marks ("") or a backslash (\\) to denote literal characters that allow the cells themselves to also contain the comma character.
```
Comment (sonnynomnom) [inline content/general/concepts/what-is-csv/what-is-csv.md:15]: Maybe move "For example, ..." to a new paragraph?

And also single tick the "" and \\. So like:

For example, some applications use quotation marks (`"`) or a backslash (`\`) to denote literal characters that allow the cells themselves to also contain the comma character.

**Comment by sonnynomnom** [comment]
Left two small comments to follow the new review process! After you make the change, I can merge!

**Comment by JeremyBarbosa** [comment]
Made the requested changes! However, do note that I had to keep the second backslash for it to render properly in markdown as can be seen here:
https://github.com/Codecademy/docs/blob/47f48b4b4fe31b836155af754228032de71e35b7/content/general/concepts/what-is-csv/what-is-csv.md
Do let me know though if this is incorrect for how you plan to render it

**Pull request merged**